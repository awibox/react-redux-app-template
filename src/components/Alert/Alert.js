import React from 'react';
import PropTypes from 'prop-types';
// Styles
import styles from './Alert.scss';

const Alert = (props) => (
  <div className={styles.alert}>
    {props.children}
  </div>
);

Alert.propTypes = {
  children: PropTypes.any,
};

export default Alert;
