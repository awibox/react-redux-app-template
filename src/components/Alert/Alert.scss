@import '../../styles/variables';

.alert {
  padding: $defaultIndent;
  background: #f47f7f;
  color: #fff;
  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
  transition: all 0.3s cubic-bezier(.25, .8, .25, 1);
  border-radius: 3px;
  margin-bottom: $defaultIndent;
}
