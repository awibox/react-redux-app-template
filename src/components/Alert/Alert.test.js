import React from 'react';
import { shallow } from 'enzyme';
import Alert from './Alert';

describe('Alert', () => {
  it('should render correctly', () => {
    const component = shallow(<Alert>Test alert</Alert>);
    expect(component).toMatchSnapshot();
  });
});
