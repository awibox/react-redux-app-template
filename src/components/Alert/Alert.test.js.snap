// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render correctly 1`] = `
<div
  className="alert"
>
  Test alert
</div>
`;
